extract/extract-handlers.ts

```diff
@@ -45,9 +45,7 @@ function handlerName(handler: AnyHandler) {
 		case HandlerTypes.Cron:
 			return pascalCase(handler.definition.name);
 		case HandlerTypes.Notification:
-			return pascalCase(
-				`${handler.definition.topicName}-${handler.definition.name}`,
-			);
+			return pascalCase(`Notification ${handler.definition.name}`);
```

Comment:
Could you leave a backwards-compatibility option here so handlers with `topicName` continue to get the same name?

Reply:
Ahh good call, yes can do